languages.yaml

```diff
@@ -1605,6 +1605,32 @@ Markdown:
       - red
     chip: "#083FA1"
   line_types: [code, comments]
+Modelica:
+  type: programming
+  ascii: |
+    {0}          #######     ########    #########
+    {0}        ########### ########################
+    {0}      #######  ###########  ########### ####
+    {0}    ######      #######     ########   #####
+    {0}    #####      ######      #######    ##### 
+    {0}    ###      ######       ######     #####  
+    {0}            ######       #####      #####   
```

Comment:
Looks like some of these lines are a bit too wide. [We enforce a 40-character maximum width at compile time](https://github.com/o2sh/onefetch/actions/runs/7636714914/job/20804507063?pr=1262#step:5:632).